osu.Framework/Graphics/Colour/SRGBColour.cs

```diff
@@ -10,65 +10,89 @@ namespace osu.Framework.Graphics.Colour
 {
     /// <summary>
     /// A wrapper struct around Color4 that takes care of converting between sRGB and linear colour spaces.
-    /// Internally this struct stores the colour in linear space, which is exposed by the Linear member.
-    /// This struct implicitly converts to sRGB space Color4 values (i.e. it can be assigned and implicitly cast)
-    /// to sRGB Color4.
+    /// Internally this struct stores the colour in linear space, which is exposed by the <see cref="Linear"/> member.
+    /// This struct converts to sRGB space by using the <see cref="SRGB"/> member.
```

Comment:
After https://github.com/ppy/osu-framework/pull/5721/commits/afa5ebc97f71a680ba0d5fa71e8972622e959d22 this is no longer correct.

```suggestion
    /// Internally this struct stores the colour in sRGB space, which is exposed by the <see cref="SRGB"/> member.
    /// This struct converts to linear space by using the <see cref="Linear"/> member.
```